.github/workflows/test-petab-test-suite.yml

```diff
@@ -28,35 +28,33 @@ jobs:
     - name: apt
       run: |
         sudo apt-get update \
-          && sudo apt-get install -y swig3.0 libatlas-base-dev \
-          && sudo ln -s /usr/bin/swig3.0 /usr/bin/swig
+          && sudo apt-get install -y \
+          swig3.0 \
+          libatlas-base-dev \
+          python3-venv
     - name: pip
       run: |
         pip3 install --upgrade --user wheel \
           && pip3 install --upgrade --user setuptools
-    - run: pip3 install pytest shyaml pytest-cov
+    - run: pip3 install pytest shyaml pytest-cov pysb petab
     - run: |
         echo ::add-path::${HOME}/.local/bin/
         echo ::add-path::${GITHUB_WORKSPACE}/tests/performance/
     # install AMICI
-    - name: Create AMICI sdist
+    - name: Install python package
       run: |
-        cd python/sdist \
-          && check_time.sh create_sdist /usr/bin/python3 setup.py sdist
-    - name: Install AMICI sdist
-      run: |
-        AMICI_PARALLEL_COMPILE=2 check_time.sh \
-            install_sdist pip3 install -v --user \
-            $(ls -t python/sdist/dist/amici-*.tar.gz | head -1)[petab,pysb]
```

Comment:
Do we still have the installation from sdist somewhere? Should be tested somewhere.

Reply:
still in use in test-large-model.yaml